ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -331,8 +354,8 @@ runWith RunNodeArgs{..} LowLevelRunNodeArgs{..} =
           (reportMetric peerMetrics)
           (NTN.mkHandlers nodeKernelArgs nodeKernel)
 
-    mkNodeToClientApps
-      :: NodeKernelArgs m (ConnectionId addrNTN) (ConnectionId addrNTC) blk
+    mkNodeToClientApps ::
+         NodeKernelArgs m (ConnectionId addrNTN) (ConnectionId addrNTC) blk
```

Comment:
The same here, let's not change the consensus style.

Reply:
Yes, thank you!